File: Programming Basics with JS/T03 - Conditional Statements/exercise/01. Sum Seconds.js
function sumSec(input) {
    let firstTime = Number(input[0]);
    let secondTime = Number(input[1]);
    let thirdTime = Number(input[2]);
    let totalTimeInSec = firstTime + secondTime + thirdTime
    let minutes = Math.floor((totalTimeInSec / 60));
    let seconds = totalTimeInSec % 60;

    if (seconds < 10) {
        console.log(`${minutes}:0${seconds}`);
    } else {
        console.log(`${minutes}:${seconds}`);
    }
}
sumSec(["22", "7", "34"]);

File: Programming Basics with Python/T03 - Conditional Statements/exercise/01. Sum Seconds.py
from math import floor

seconds_first = int(input())
seconds_second = int(input())
seconds_third = int(input())

total_seconds = seconds_first + seconds_second + seconds_third

minutes = total_seconds // 60
seconds = total_seconds % 60
minutes = floor(minutes)

if seconds < 10:
    print(f'{minutes}:0{seconds}')
else:
    print(f'{minutes}:{seconds}')